swift/Tests/GildedRoseTests/GildedRoseTests.swift

```diff
@@ -3,16 +3,14 @@ import XCTest
 
 class GildedRoseTests: XCTestCase {
 
-    func testFoo() {
+    func testFoo() throws {
         let items = [Item(name: "foo", sellIn: 0, quality: 0)]
-        let app = GildedRose(items: items);
-        app.updateQuality();
-        XCTAssertEqual("fixme", app.items[0].name);
+        let app = GildedRose(items: items)
+        app.updateQuality()
+        XCTAssertEqual(app.items[0].name, "fixme")
```

Comment:
Most xUnit equality assertions follow the JUnit convention of putting the expected value on the left, clearly identifying it as the expectation in the error message.

This is not true in XCUnit. Sticking with expectation-on-left leads to this message:

> ("fixme") is not equal to ("foo")

This is a little awkward to read, so I encourage folks to switch the arguments. This yields

> ("foo") is not equal to ("fixme")

which is just a little easier to read, left to right.